packages/forms-web-app/__tests__/unit/controllers/examination/check-submission-item/utils/summary-list-item/submission-item.test.js

```diff
@@ -9,7 +9,20 @@ describe('controllers/examination/check-submission-item/utils/summary-list-item/
 				const mockSubmissionItem = {
 					submissionItem: 'submission item'
 				};
-				const result = getSummaryListItemSubmissionItem(mockSubmissionItem);
+
+				const session = {
```

Comment:
this should not be used. 

it should call a mock / stub. 

showDeadlineItemChangeUrl should be a mock.

Reply:
this test is also missing the conditional test which is what the mock would also allow us to trigger.